databases/week1/homework.sql

```diff
@@ -0,0 +1,130 @@
+
+--- task 1 ---
+SELECT
+  COUNT(id)
```

Comment:
COUNT(Id) as task_amount
_Pointer_
Add a column name to all columns created by you